plugins/wasi_nn_rust/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "wasi_nn_rust"
+version = "0.0.1"
+edition = "2021"
```

Comment:
Should we use `2023` here?

Reply:
It seems that the 2023 edition is not available. The latest edition of Rust that can be used is 2024, but it is currently still in [beta](https://doc.rust-lang.org/edition-guide/rust-2024/index.html). Here, the previous usable version, 2021, is kept to resolve this issue.